src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -220,6 +220,7 @@ private SuggestedActionSet InlineActions(SuggestedActionSet actionSet)
                         }
 
                         newActions.AddRange(childActionSets[0].Actions);
+                        continue;
                     }
 
                     newActions.Add(action);
```

Comment:
Why not remove the `continue` and wrap this line in an `else { }`?  It (should) accomplish the same thing and it's easier to look at.